src/main/java/nl/knmi/geoweb/backend/aviation/AirportStore.java

```diff
@@ -47,7 +47,7 @@ public AirportStore(@Value(value = "${productstorelocation}") String productstor
 	public void initStore() throws IOException {
 		this.airportInfos=new HashMap<String,AirportInfo>();
 		File fn=new File(this.directory+"/"+this.airportFile);
-		Debug.println("fn:"+fn);
+		Debug.errprintln("fn:"+fn);
```

Comment:
For the time being I think we should use nl.knmi.adaguc.tools.Debug.errprintln() instead of System.err/out.println().
Fixed in this PR.
Later we might need to look at LOG4J or on of it's derivatives.